pkg/osquery/tables/dev_table_tooling/table_test.go

```diff
@@ -0,0 +1,44 @@
+package dev_table_tooling
+
+import (
+	"context"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_generate(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name           string
+		commandName    []string
+		expectedResult []map[string]string
+	}{
+		{
+			name: "no command name",
+		},
+		{
+			name:        "malware",
+			commandName: []string{"ransomware.exe"},
+		},
```

Comment:
Might be worth adding some kind of happy path testing. Which raises the question of what command. I suspect `echo hello` might be okay, can't quickly double check that's not a shell built on linux, but it should work on mac and windows.